src/tool_writeout.c

```diff
@@ -380,21 +380,26 @@ void ourWriteOut(const char *writeinfo, struct per_transfer *per,
             }
           }
           if(!match) {
-            fprintf(stderr, "curl: unknown --write-out variable: '%s'\n", ptr);
+            fprintf(stderr, "curl: unknown --write-out variable: '%.*s'\n",
+                    (int)vlen, ptr);
           }
           ptr = end + 1; /* pass the end */
```

Comment:
typo here as well, suggest remove comment, otherwise lgtm

Reply:
But the comment is unrelated to this change, I prefer not to modify it (now)